docs/developers/contributing.md

```diff
@@ -44,14 +44,14 @@ More information can be found in [gulpfile.js](https://github.com/chartjs/Chart.
 
 ### Image-Based Tests
 
-Some display-related functionality is difficult to test via typical Jasmine unit test asserts. For this functionality, we have image-based tests that assert that the chart is drawn pixel-for-pixel matching an expected image.
+Some display-related functionality is difficult to test via typical Jasmine units. For this reason, we introduced image-based tests ([#3988](https://github.com/chartjs/Chart.js/pull/3988) and [#5777](https://github.com/chartjs/Chart.js/pull/5777)) that assert that a chart is drawn pixel-for-pixel matching an expected image.
```

Comment:
Still not native, but "that assert that" sounds wrong, is it? "to assert that"? "for asserting that"?

Reply:
I'm not sure that it's wrong, but it does sound slightly awkward to have "that" appear twice in such close proximity. I'll change to "to assert that"